### ISSUE TITLE
jinja tags are different than HTML tags & body text

### ISSUE THREAD

**Issue by wraybowling** [mitsuhiko/jinja2-tmbundle#6]
Look at the way django templates are highlighted. https://github.com/squ1b3r/Djaneiro

I think I would prefer jinja template highlighting to behave the same way. html tags are one color and template tags are another.

Variables between {{ braces }} are also not body text and should not be colored the same way.

Note: that package has quirks of its own however, such as not highlighting curly-braces between double-quotes

**Comment by jacobsvante** [comment]
I agree. With the way this plugin is setup now I prefer using regular HTML syntax where every Jinja-tag becomes one white and the HTML is highlighted as usual.
